main.py

```diff
@@ -1,22 +1,40 @@
-from flask import Flask, request
-from flask_json import FlaskJSON
+import json
+from flask import Flask, request, make_response
 
-app = Flask(__name__)
-FlaskJSON(app)
+def create_app():
+    app = Flask(__name__)
+    return app
 
-@app.route("/")
-def hello():
-    return "Hello World!"
+# if __name__ == '__main__':
+app = create_app()
+app.run()
 
 @app.route('/login', methods=['POST'])
 def login():
+    try:
+        if request.headers['Content-Type'] != 'application/json':
+            message = 'Content-Type not json'
+            http_code = 400
+            return message, http_code
+        
+        data = json.loads(request.get_data())
+   
+    except:
```

Comment:
It's a good idea to specify the exception type here which would also enable you to communicate the error to the user.

Reply:
I've tried to put here `except JSONDecodeError as err`, but it raised Error:  'JSONDecodeError is not defined'. But when I put not a json data without exception, program raised JSONDecodeError

Reply:
Dealt with it.